islandora_binary_object.module

```diff
@@ -96,7 +96,10 @@ function islandora_binary_object_xml_form_builder_forms() {
 /**
  * Implements hook_islandora_ingest_steps().
  */
+ // @codingStandardIgnoreStart
+ // Codesniffer doesn't like the mixed cases in the content model PID and wants a different form.
```

Comment:
This and the similar comments seem rather unnecessary. This is as standard as a thing gets and there is automation around for explanation.